src/tests/api/ApiTestSteps.ts

```diff
@@ -420,3 +422,22 @@ export async function initiateUserInfo(docSelectionData: DocSelectionData, sessi
 	expect(userInfoResponse.status).toBe(202);
 
 }
+
+export async function abortPost(sessionId: string): Promise<AxiosResponse<string>> {
```

Comment:
We should already have an abort function that was being used in the old tests.

line 359